lib/lightning_web/router.ex

```diff
@@ -77,15 +77,35 @@ defmodule LightningWeb.Router do
   scope "/", LightningWeb do
     pipe_through [:browser, :require_authenticated_user]
 
-    get "/users/two-factor/app", UserTOTPController, :new
-    post "/users/two-factor/app", UserTOTPController, :create
+    get "/users/two-factor", UserTOTPController, :new
+    post "/users/two-factor", UserTOTPController, :create
 
     get "/profile/confirm_email/:token",
         UserConfirmationController,
         :confirm_email
 
     get "/download/yaml", DownloadsController, :download_project_yaml
 
+    live_session :auth, on_mount: LightningWeb.InitAssigns do
+      live "/auth/confirm_access", ReAuthenticateLive.New, :new
+    end
```

Comment:
Why is this in it's own `live_session`? Is it because we need to ensure we're in a browser redirect to apply a new cookie?